ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -36,11 +36,17 @@
   file: path=/tmp/sync_static.py state=absent
 
 - name: Get os_local_settings_path
-  shell: "find {{ horizon_virtual_env }} -type d -path /sys -prune -o -name '*local_settings.d' 2>/dev/null | grep -E 'openstack_dashboard|horizon'"
+  find:
+    paths: /openstack/venvs/
```

Comment:
why this path is hardcoded ? aren't we using a env_path variable in global.yaml ?
we removed this variable {{ horizon_virtual_env }} ?

Reply:
value of {{ horizon_virtual_env }} is like /openstack/vens/horizon-<version>/bin/activate
local_settings.d present into /openstack/vens/horizon-<version> hence we have hardcoded.